### DISCUSSION TITLE
Q: netlify部署显示Not initialized || Q: netlify deployment shows Not initialized

### DISCUSSION THREAD

**Discussion by lfyszy** [walinejs/waline#2064] [Q&A] [answered]
### Describe the bug | 问题描述

怎么解决呀？

### Website URL | 问题网站

https://boisterous-jelly-e504a4.netlify.app/.netlify/functions/comment

### Where your waline deploy? | 服务部署在哪里？

Netlify

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

How to solve it?

### Website URL | Problem Website

https://boisterous-jelly-e504a4.netlify.app/.netlify/functions/comment

### Where your waline deploy? | Where is the service deployed?

Netlify

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
https://github.com/orgs/walinejs/discussions?discussions_q=Not+initialized
https://github.com/walinejs/waline/issues?q=is%3Aissue+Not+initialized+is%3Aclosed
